vue/components/ui/molecules/filter/filter.vue

```diff
@@ -19,13 +19,17 @@
                 v-bind:get-item-url="getItemUrl"
                 v-on:click="onLineupClick"
             >
-                <slot v-bind:name="slot" v-for="slot in Object.keys($slots)" v-bind:slot="slot" />
+                <slot
+                    v-bind:name="slot"
+                    v-for="slot in scopedSlots"
+                    v-bind:slot="slot.replace('lineup-', '')"
+                />
                 <template
-                    v-for="slot in Object.keys($scopedSlots)"
-                    v-bind:slot="slot"
+                    v-for="slot in Object.keys(this.$scopedSlots)"

```

Comment:
Why `this`? Did you mean to use your new computed? In that case, the name is wrong.